schema.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
     date_of_birth date,
     escape_attemps int,
     neutered boolean,
-    weight_kg decimal 
+    weight_kg decimal,
+    species varchar(100) 
```

Comment:
- [ ] [REQUIRED] Great job trying to add a new column called `species` with a string type. However, you need to alter an existing table in order to be able to do this. Let me give an example of this:

`ALTER TABLE countries ADD germany varchar(100);`

Kindly fix this issue before requesting another review 👍🏽